### ISSUE TITLE
testGreedyVertexColoring fails for 0.10.4

### ISSUE THREAD

**Issue by Apteryks** [igraph/python-igraph#632]
**Describe the bug**
When running the test suite, I observe:
```
=================================== FAILURES ===================================
_________________ VertexColoringTests.testGreedyVertexColoring _________________

self = <tests.test_coloring.VertexColoringTests testMethod=testGreedyVertexColoring>

    def testGreedyVertexColoring(self):
        g = Graph.Famous("petersen")
    
>       col = g.vertex_coloring_greedy()
E       igraph._igraph.InternalError: Error at src/misc/coloring.c:300: Invalid heuristic for greedy vertex coloring. -- Invalid value

tests/test_coloring.py:16: InternalError
=========================== short test summary info ============================
FAILED tests/test_coloring.py::VertexColoringTests::testGreedyVertexColoring
================== 1 failed, 448 passed, 17 skipped in 5.85s ===================
```

**To reproduce**
Build the package from source in a clean environment using the following dependencies:
```
igraph@0.10.4 python-pytest@6.2.5 python-texttable@1.6.4
```

**Version information**
Which version of `python-igraph` 0.10.4 from git.

**Comment by ntamas** [comment]
I just removed the 0.10.4 tag a while ago because of this bug; the fix is in b6ebd8e and it will be the "real" 0.10.4. (Note that there was no official tarball pushed to PyPI yet).

**Comment by Apteryks** [comment]
Ok! In the future, please treat tags as immutable;  adding a new 0.10.5 tag instead would make it easier on everyone :-).
